Update mobilebroadbandmodem_isresetsupported.md

samyun opened the pull request:
Broken link

samyun committed:
Update mobilebroadbandmodem_isresetsupported.md

Broken link

mikematteson commented:
@stevewhims : PR **passed** acceptance checklist. Please evaluate further and approve/merge if appropriate. Thanks!

stevewhims committed:
Tweaks...

Don't need 2 links just within a few words of each other; making the true/false language consistent; removing "will", because that's our standard (probably makes it sound like it "will" if a certain further condition obtains whereas we really mean that it **does** under the condition we just gave).

stevewhims reviewed (approved):
LGTM!
